serverless.yml

```diff
@@ -62,18 +66,18 @@ custom:
           contentType: application/json,
           schema: '${file(src/global/models/user-settings.json):patch.response}',
         }
-      - {
-          name: PostDogesOnTrialDogeImagesRequest,
-          description: 'The request for postDogesOnTrialDogeImages.',
-          contentType: application/json,
-          schema: '${file(src/doges-on-trial/models/doge-images.json):post.request}',
-        }
       - {
           name: PostDogesOnTrialDogeImagesResponse,
           description: 'The response for postDogesOnTrialDogeImages.',
           contentType: application/json,
           schema: '${file(src/doges-on-trial/models/doge-images.json):post.response}',
         }
+      - {
+          name: PostEmailSenderRequest,
```

Comment:
My thinking is this is basically an example service. I played with making some generic and extensible but I think it just makes things more complicated if anyone has event handling that is mildly out of the ordinary. So I mostly set this up so that people can quickly copy/paste and edit to fit their own use case